src/pages/TransactionHistory/hooks/useSearch.tsx

```diff
@@ -2,11 +2,11 @@ import type { Asset } from '@shapeshiftoss/asset-service'
 import debounce from 'lodash/debounce'
 import { useCallback, useState } from 'react'
 import { filterAssetsBySearchTerm } from 'components/AssetSearch/helpers/filterAssetsBySearchTerm/filterAssetsBySearchTerm'
-import { selectAssetsByMarketCap } from 'state/slices/selectors'
+import { selectAssetsByMarketCapAndName } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
 export const useSearch = () => {
-  const assets = useAppSelector(selectAssetsByMarketCap)
+  const assets = useAppSelector(selectAssetsByMarketCapAndName)
```

Comment:
Do we still need the `useSearch()` there doing any sorting at all here anymore? `selectAssets` without sorting seems like it is enough, given we don't really need sorting here?

Reply:
No we don't. We don't even need assets altogether - to be tackled in a follow up. TODOs added in 1dc15b1f17fe8486ed0a35e23d6d080b94ecc765. Switched to `selectAssets` and memoized `Object.values()` of `assets` for the time being, to be updated to AssetId right on the next PR not to clutter this